### PR TITLE
Raising svc_connected and svc_disconnected for all nodes in a cluster

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id: Tendrl/gluster-integration#694
bugzilla: 1616215, 1616208

Signed-off-by: GowthamShanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @nthomas-redhat please review

File: tendrl/gluster_integration/message/callback.py [hunk 1]
```diff
@@ -69,15 +69,20 @@ def quorum_regained(self, event):
         native_event.save()
 
     def svc_connected(self, event):
-        context = "svc_connection|" + event['message']['svc_name']
+        context = "svc_connection|" + "%s_%s" % (
+            event['message']['svc_name'],
+            NS.node_context.fqdn
+        )
```
Comment (shtripat) [inline tendrl/gluster_integration/message/callback.py]: If I understand correctly the context in this scenario would be of form `svc_connection|glusterd_{node-fqdn}` and if the volume name present in message, the context would be `svc_connection|glusterd_{node-fqdn}{volume-name}`.
The question is how are we processing this context? Some comment here would be helpful if you explain how the context would be tackled.

Comment (GowthamShanmugam) [reply tendrl/gluster_integration/message/callback.py]: All callback functions will save the native events into etcd using  `NS.gluster.objects.NativeEvents` object. The provisioner node will call process event function in each gluster-integration sync https://github.com/Tendrl/gluster-integration/blob/38d5d66c1d0f2982edbc2702a9d87e1d20df7864/tendrl/gluster_integration/sds_sync/__init__.py#L263 to process the native events.

Then process event function https://github.com/Tendrl/gluster-integration/blob/38d5d66c1d0f2982edbc2702a9d87e1d20df7864/tendrl/gluster_integration/message/process_events.py#L9 will load all native objects and push that object into the message socket as alerts.

Context is used as `plugin_instance` to a different one alert from another.

**Comment by anmolsachan** [comment]
LGTM

**Pull request merged**